samples/highcharts/export-data/export-post-request/demo.js

```diff
@@ -0,0 +1,40 @@
+$(function () {
```

Comment:
The sample not work, jQuery is not included in the sample. Since the purpose of this demo is to show how to do it with legacy browsers, just add jquery in a script tag in `demo.html`. And remember to test that it's actually working.

Reply:
I've added the jQuery tag. Highcharts-Utils however have a problem with sending that request (by Content Security Policy directive) so it doesn't work when ran locally with utils, but everything works in jsfiddle.